script/options.js

```diff
@@ -36,19 +36,23 @@ function gatherFromCommandLine() {
 }
 
 function applyDefaultOptions(options) {
+  const contentPagesRoot = options['content-directory'];
+  const contentRoot = path.join(contentPagesRoot, '../');
```

Comment:
That command arg should probably be updated to reference the whole repo instead of just the pages directory.